tls/_common/enums.py

```diff
@@ -45,6 +45,8 @@ class HandshakeType(Enum):
     CERTIFICATE_VERIFY = 15
     CLIENT_KEY_EXCHANGE = 16
     FINISHED = 20
+    CERTIFICATE_URL = 21
+    CERTIFICATE_STATUS = 22
```

Comment:
Yes!! I'll play around with that next, blends in well with the #106 branch I was thinking of tackling.